modules/ROOT/pages/dw-core-namespaces.adoc

```diff
@@ -1,14 +1,11 @@
 = Core Namespaces (dw::Core)
 
+[%header, cols="1,2a,3a"]
 |===
 | Namespace | Definition | Description
 
 | xsi
-| `xsi = http://www.w3.org/2001/XMLSchema-instance`
+| `+xsi = http://www.w3.org/2001/XMLSchema-instance+`
```

Comment:
TODO (DEV): is it ok to make this change (`+xsi = http://www.w3.org/2001/XMLSchema-instance+`) in Core.dwl? 

TODO (DUKE): add to Core.dwl if approved.

Reply:
NO (REMAINS MANUAL): not possible, it would seem, as this is part of the ns definition:
ns xsi http://www.w3.org/2001/XMLSchema-instance